Integrations/GitHub/GitHub.py

```diff
@@ -265,7 +642,266 @@ def get_download_count():
+            'Deletions': pr_file.get('deletions'),
+            'Changes': pr_file.get('changes')
+        }
+        for pr_file in response
+    ]
+    ec_object = {
+        'Number': pull_number,
+        'File': formatted_pr_files
+    }
+    ec = {
+        'GitHub.PR(val.Number === obj.Number)': ec_object
+    }
+    human_readable = tableToMarkdown(f'Pull Request Files for #{pull_number}', formatted_pr_files, removeNull=True)
+    return_outputs(readable_output=human_readable, outputs=ec, raw_response=response)
+
+
+def list_issue_comments_command():
+    args = demisto.args()
+    issue_number = args.get('issue_number')
+    response = list_issue_comments(issue_number)
+
+    ec_object = [format_comment_outputs(comment, issue_number) for comment in response]
+    ec = {
+        'GitHub.Comment(val.IssueNumber === obj.IssueNumber && val.ID === obj.ID)': ec_object
+    }
+    human_readable = tableToMarkdown('Comments', ec_object, removeNull=True)
+    return_outputs(readable_output=human_readable, outputs=ec, raw_response=response)
+
+
+def create_comment_command():
+    args = demisto.args()
+    issue_number = args.get('issue_number')
+    body = args.get('body')
+    response = create_comment(issue_number, body)
+
+    ec_object = format_comment_outputs(response, issue_number)
+    ec = {
+        'GitHub.Comment(val.IssueNumber === obj.IssueNumber && val.ID === obj.ID)': ec_object
+    }
+    human_readable = tableToMarkdown('Created Comment', ec_object, removeNull=True)
+    return_outputs(readable_output=human_readable, outputs=ec, raw_response=response)
+
+
+def request_review_command():
+    args = demisto.args()
+    pull_number = args.get('pull_number')
+    reviewers = argToList(args.get('reviewers'))
+    response = request_review(pull_number, reviewers)
+
+    requested_reviewers = response.get('requested_reviewers', [])
+    formatted_requested_reviewers = [format_user_outputs(reviewer) for reviewer in requested_reviewers]
+    ec_object = {
+        'Number': response.get('number'),
+        'RequestedReviewer': formatted_requested_reviewers
+    }
+    ec = {
+        'GitHub.PR(val.Number === obj.Number)': ec_object
+    }
+    human_readable = tableToMarkdown(f'Requested Reviewers for #{response.get("number")}',
+                                     formatted_requested_reviewers, removeNull=True)
+    return_outputs(readable_output=human_readable, outputs=ec, raw_response=response)
+
+
+def get_team_membership_command():
+    args = demisto.args()
+    team_id = 0
+    try:
+        team_id = int(args.get('team_id'))
+    except ValueError as e:
+        return_error('"team_id" command argument must be an integer value.', e)
+    user_name = args.get('user_name')
+    response = get_team_membership(team_id, user_name)
+
+    ec_object = {
+        'Role': response.get('role'),
+        'State': response.get('state'),
+        'ID': team_id
+    }
+    ec = {
+        'GitHub.Team': ec_object
```

Comment:
remove ec outputs

Reply:
changed the outputs in the end... not removed